src/luascript.h

```diff
@@ -1182,6 +1182,9 @@ class LuaScriptInterface
 
 		static int luaItemTypeIsStoreItem(lua_State* L);
 
+		static int luaItemTypeGetDefaultDurationMin(lua_State* L);
```

Comment:
Since the repository where I made these changes was deleted, I cannot fix these details, but in case of being merged, of course I will fix the indentation and tabs correctly, even you could do it yourself in any case.